catalog/src/test/java/org/killbill/billing/catalog/TestCatalogService.java

```diff
@@ -33,7 +33,7 @@ public void testCatalogServiceDirectory() throws ServiceException, CatalogApiExc
         final DefaultCatalogService service = new DefaultCatalogService(new CatalogConfig() {
             @Override
             public String getCatalogURI() {
-                return "file:src/test/resources/versionedCatalog";
+                return "file:catalog/src/test/resources/versionedCatalog";
             }
```

Comment:
ohh, interesting! You are right, the test without "catalog/" is failing on IntelliJ but it works fine on mvn command. Let me change it back.
Thank you!